scantools/run_navvis_to_capture.py

```diff
@@ -134,6 +133,55 @@ def run(input_path: Path, capture: Capture, tiles_format: str, session_id: Optio
                     pose = get_pose(nv, upright, frame_id, camera_id, tile_id)
                     trajectory[timestamp_us, sensor_id] = pose
 
+    if export_trace and export_as_rig:
+        # Add "trace" to the rig with identity pose.
+        rigs[rig_id, "trace"] = Pose()
+
+        # Add "trace" as a sensor.
+        sensors['trace'] = create_sensor('trace', name='Mapping path')
+
+        qvec, tvec = nv._imu["orientation"], nv._imu["position"]
+        camhead_from_imu = Pose(r=qvec, t=tvec)
+        imu_from_camhead = camhead_from_imu.inverse()
+
+        # Rig is in cam0 frame.
+        cam0 = nv.get_cameras()["cam0"]
+        qvec, tvec = cam0["orientation"], cam0["position"]
+        camhead_from_rig = Pose(r=qvec, t=tvec)
+
+        for trace in nv.get_trace():
+            timestamp_us = int(trace["nsecs"]) // 1_000  # convert from ns to us
+
+            # world_from_imu (trace.csv contains the IMU's poses)
+            qvec = np.array([trace["ori_w"], trace["ori_x"], trace["ori_y"], trace["ori_z"]], dtype=float)
+            tvec = np.array([trace["x"], trace["y"], trace["z"]], dtype=float)
+            world_from_imu = Pose(r=qvec, t=tvec)
+
+            # Apply the transformation to the first tile's pose.
```

Comment:
Can't we use here rig_from_world defined above?

Reply:
The `rig_from_world` above is defined only in the `frame_id=0`, here I need to update it every sample in the trace, so think it as it would have a subindex: `word_from_rig_t` where `t` is one sample in the trace. Otherwise, I will always get the same static pose.